modules/assim.batch/R/pda.mcmc.bs.R

```diff
@@ -29,7 +29,7 @@ pda.mcmc.bs <- function(settings, params.id=NULL, param.names=NULL, prior.id=NUL
                   adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot, burnin=burnin)
  
    burnin <- ifelse(!is.null(settings$assim.batch$burnin), 
-                    settings$assim.batch$burnin, 
+                    as.numeric(settings$assim.batch$burnin), 
                     ceiling(min(2000,0.2*settings$assim.batch$iter)))
```

Comment:
I'm fine with that default for now until we get code that tests for burn-in (guess that's the one thing in the to do list that didnt' get done)